src/plugins/ua-parse.js

```diff
@@ -8,7 +8,12 @@ export default function (context, inject) {
   } else if (typeof navigator !== 'undefined') {
     userAgent = navigator.userAgent
   }
-  const ua = useragent.parse(userAgent)
+  var ua
```

Comment:
I'm not sure why our linter isn't complaining about this line, we have the `no-var` rule enabled :confused: 

```suggestion
  let ua
```

Reply:
And I will make the necessary changes to `use-browser-detection.js` and index.d.ts too. Thanks for the heads up.